controllers/storagecluster/cephconfig.go

```diff
@@ -36,6 +36,7 @@ mon_osd_nearfull_ratio = .75
 mon_max_pg_per_osd = 600
 mon_pg_warn_max_object_skew = 0
 mon_data_avail_warn = 15
+bluestore_prefer_deferred_size_hdd = 0
```

Comment:
Looks like this change will affect all customers whether DR or not.  Is that the expectation?

Reply:
Also, since #2053 is creating all the OSDs as SSDs, do we still need this? I'm not clear on the effect of this setting.